src/components/PeopleTableBody/PeopleTableBody.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from 'react';
+import cn from 'classnames';
```

Comment:
It is better not to shorten the names to unreadable ones (`cn` -> `classNames`).
This applies to all code.